tests/unit/_memory_storage_client/test_request_queue_client.py

```diff
@@ -93,19 +93,20 @@ async def test_list_head(request_queue_client: RequestQueueClient) -> None:
 
 
 async def test_request_state_serialization(request_queue_client: RequestQueueClient) -> None:
-    request = Request.from_url('https://crawlee.dev')
+    request = Request.from_url('https://crawlee.dev', payload=b'test')
     request.state = RequestState.UNPROCESSED
 
     await request_queue_client.add_request(request)
 
     result = await request_queue_client.list_head()
-
     assert len(result.items) == 1
     assert result.items[0] == request
 
     got_request = await request_queue_client.get_request(request.id)
 
     assert request == got_request
+    assert request.payload == got_request.payload
+    assert got_request.payload != b"b'test'"
```

Comment:
I believe this should be enough, correct?

```suggestion
    assert request.payload == got_request.payload
```